packages/salesforcedx-vscode-apex/src/constants.ts

```diff
@@ -1,97 +1,21 @@
 /*
- * Copyright (c) 2017, salesforce.com, inc.
+ * Copyright (c) 2023, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import * as path from 'path';
 import * as vscode from 'vscode';
-import { nls } from './messages';
 
 export const DEBUGGER_LINE_BREAKPOINTS = 'debugger/lineBreakpoints';
 export const DEBUGGER_EXCEPTION_BREAKPOINTS = 'debugger/exceptionBreakpoints';
 
-export const LIGHT_BLUE_BUTTON = path.join(
```

Comment:
So the issue with the icons was the code assumed a location of the source and found the resources directory from the current file location.  That doesn't work with bundling b/c the source moves from this dir to the minified js file. I created a new class to handle it in a sane way instead of trying to hack it together as a constant value.